### PR TITLE
Canonical CNA: add CVE-2020-8835 / linux kernel / USN-4313-1

### PR DESCRIPTION
**Pull request by stevebeattie**
Signed-off-by: Steve Beattie <steve.beattie@canonical.com>

### CODE REVIEW

**Comment by cve-team** [comment]
stevebeattie is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2020/8xxx/CVE-2020-8835.json [hunk 1]
```diff
@@ -1,18 +1,123 @@
...
     "CVE_data_meta": {
+        "ASSIGNER": "security@ubuntu.com",
+        "DATE_PUBLIC": "2020-03-30T16:00:00.000Z",
         "ID": "CVE-2020-8835",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Linux kernel bpf verifier vulnerability"
     },
+    "affects": {
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8835.json]: Missing information in vendor_name, product_name, and version_data fields.

Comment (stevebeattie) [reply 2020/8xxx/CVE-2020-8835.json:9]: Thanks, I've added those details to the json and rebased the branch. Alas, I didn't realize that github wouldn't show the differences between the rebased commit versions (gitlab, which I'm more familiar with, will do that). Sorry about that.

**Comment by stevebeattie** [comment]
For the record, the upstream kernel has not fixed this issue yet, but I would expect it to be addressed in the next stable versions of 5.4, 5.5, and 5.6, as it has landed in the net-next tree.

**Comment by stevebeattie** [comment]
I have now added the upstream fix information.

**Pull request merged**